backend/src/services/auth/auth.service.ts

```diff
@@ -21,6 +23,17 @@ class Auth {
       password: await encrypt(password),
     });
   }
+
+  public async signIn(payload: SignInPayload): Promise<User | null> {
+    const { password, email } = payload;
+    const user = await this.#userRepository.getByEmail(email);
+
+    if (!user || !(await cryptCompare(password, user.password))){
+      throw new HttpError({ status: HttpCode.NOT_FOUND, message: 'User not found' })
```

Comment:
take a look at this enum https://github.com/BinaryStudioAcademy/bsa-2021-jabber/pull/34#pullrequestreview-719550200